datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/service_checks.py

```diff
@@ -102,6 +116,9 @@ def service_checks():
             if integration is None or not isinstance(integration, string_types):
                 file_failed = True
                 display_queue.append((echo_failure, '  required non-null string: integration'))
+            if integration != decoded_manifest['display_name']:
```

Comment:
```suggestion
            if integration != decoded_manifest['display_name'].lower():
```
:)

Reply:
Actually it seems that currently the display is also widely used, so, I guess we can keep the title case.

Reply:
cc @ofek